packages/mobile/src/tokens/saga.ts

```diff
@@ -73,15 +77,19 @@ export function tokenFetchFactory({ actionName, token, actionCreator, tag }: Tok
       const tokenContract = yield call(getTokenContract, token)
       const balanceInWei: BigNumber = yield call([tokenContract, tokenContract.balanceOf], account)
       const balance: BigNumber = yield call(convertFromContractDecimals, balanceInWei, token)
-      ValoraAnalytics.track(
-        AppEvents.fetch_balance,
-        token === CURRENCY_ENUM.DOLLAR
-          ? {
-              dollarBalance: balance.toString(),
-            }
-          : { goldBalance: balance.toString() }
-      )
+
       yield put(actionCreator(balance.toString()))
```

Comment:
Should setting the balance also be skipped when it is too large, instead of hiding it?

The concern with hiding it with the above check with formattedValue is that there are other checks that may incorrectly succeed (whether someone has sufficient balance in the Send flow, for example)

Reply:
Good point. I guess that implies that we would continue to show the last fetched balance instead of the erroneous one, which is a more elegant solution.

I'll make this change, thanks!